New Personal Website/contact.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <h3>Contact details</h3>
+    <ul>
+      <li><em><strong>Phone Number</strong></em>:7022814594</li>
+      <li><em><strong>Email</strong></em>:rahul4132@gmail.com</li>
```

Comment:
probably don't want to put this out in public lol unless you want spam porn

Reply:
lol yeah makes sense :)